RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate SetUsernameView to Typescript

## Proposed changes

Migrate SetUsernameView to Typescript

## Issue(s)	


## How to test or reproduce

- Create a new account with the username as `test+username`, you need to write the username with `+`
- Then, after validating your email, do the login, you should be moved to another screen with the title `Username` and button `Register`. This is the view migrated

## Screenshots

## Types of changes




## Checklist



## Further comments
